test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java

```diff
@@ -279,6 +300,7 @@ public Object createTestInstance(TestInstanceFactoryContext factoryContext, Exte
                     try {
                         launcher.start();
                     } catch (IOException e) {
+                        launcher.close();
```

Comment:
What if close fails? :) I'm ok with that change but is that with an understanding that launcher.close() will always work even of start failed?